pkg/providers/common/common.go

```diff
@@ -67,17 +69,18 @@ func GenerateSSHAuthKey(writer filewriter.FileWriter) (string, error) {
 		return "", fmt.Errorf("generating ssh key pair: %v", err)
 	}
 
-	logger.Info(fmt.Sprintf(
-		"Private key saved to %[1]s. Use 'ssh -i %[1]s <username>@<Node-IP-Address>' to login to your cluster node",
-		privateKeyPath,
-	))
+	logger.Info("Private key saved", "sshCommand", fmt.Sprintf("ssh -i %s <username>@<Node-IP-Address>", privateKeyPath))
```

Comment:
nit: wondering if we should just print the path to the ssh key as opposed to the command, since this is now generic and not for an specific provider.

Reply:
Logging this way also creates a less than desirable user experience because it dumps a json string. Ideally users aren't looking at JSON strings in the console (I know we have a bunch of that today).

Reply:
but this cmd is also generic. it applies to all providers

Reply:
1. this cmd is generic across all providers
2. I don't have strong preference on whether using json or not. I will just remove this change in this PR